bin/analyze-css.js

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const execSync = require( 'child_process' ).execSync;
+const fs = require( 'fs-extra' );
+const path = require( 'path' );
+
+const PROJECT_DIRECTORY = path.resolve( __dirname, '..' );
+const TEMP_DIRECTORY = path.resolve( PROJECT_DIRECTORY, 'temp' );
+const DEBUG_BUNDLE = path.join( TEMP_DIRECTORY, 'styles.debug.css' );
+
+console.log( `> Cleaning up ${ TEMP_DIRECTORY } directory` );
+
+fs.removeSync( TEMP_DIRECTORY );
+
+console.log( '> Generating CSS bundle with source comments' );
+
+execSync( `node_modules/node-sass/bin/node-sass --include-path client --source-comments assets/stylesheets/style.scss ${ DEBUG_BUNDLE }` );
```

Comment:
Maybe I'm missing something, so I'm sorry in advance.

1. Aren't we missing the editor.css, directly.css and others?
2. How will be keep that in sync with whatever changes are going to happen in `package.json`, why not just use `npm run build-css` or take the command from there?

Reply:
> Aren't we missing the editor.css, directly.css and others?

Good catch, I forgot them since they are handled with a separate script. 

> why not just use npm run build-css or take the command from there?

I took the command from there, but forgot to include the `autoprefixer`. I'll correct that. Apart from that, I can't really run the `build-css` script since I need to generate stylesheets with the `--source-comments` parameter. Maybe you can think of a better approach?